src/test/java/com/cloudant/tests/DesignDocumentsTest.java

```diff
@@ -52,38 +55,61 @@
 
     private static Database db;
     private CloudantClient account;
+    private File rootDesignDir;
+    private File designDocExample;
+    private DesignDocumentManager designManager;
 
 
     @Before
     public void setUp() {
         account = clientResource.get();
         db = dbResource.get();
+        rootDesignDir = new File(System.getProperty("user.dir")
+                + "/src/test/resources/design-files");
+        designDocExample = new File(String.format("%s/example_design_doc.js", rootDesignDir));
+        designManager = db.getDesignDocumentManager();
     }
 
     @Test
-    public void designDocSync() {
-        DesignDocument designDoc = db.design().getFromDesk("example");
-        db.design().synchronizeWithDb(designDoc);
+    public void designDocSync() throws MalformedURLException {
+        DesignDocument designDoc = DesignDocumentManager.fromFile(designDocExample);
+
+        db.getDesignDocumentManager().put(designDoc);
     }
 
     @Test
-    public void designDocCompare() {
-        DesignDocument designDoc1 = db.design().getFromDesk("example");
-        db.design().synchronizeWithDb(designDoc1);
+    public void designDocCompare() throws MalformedURLException {
+        //Use File or String for fromDir parameter?
+        DesignDocument designDoc1 = DesignDocumentManager.fromFile(designDocExample);
+        designManager.put(designDoc1);
 
-        DesignDocument designDoc11 = db.design().getFromDb("_design/example");
+        DesignDocument designDoc11 = db.getDesignDocumentManager().get("_design/example");
 
         assertEquals(designDoc1, designDoc11);
     }
 
     @Test
     public void designDocs() {
-        List<DesignDocument> designDocs = db.design().getAllFromDesk();
-        db.syncDesignDocsWithDb();
+        List<DesignDocument> designDocs = DesignDocumentManager.fromDirectory(rootDesignDir);
+        //db.syncDesignDocsWithDb();
+        DesignDocument[] docArray = designDocs.toArray(new DesignDocument[designDocs.size()]);
+        designManager.put(docArray);
+        assertThat(designDocs.size(), not(0));
+    }
+
+    @Test
+    public void updateDesignDocs() {
+        List<DesignDocument> designDocs = DesignDocumentManager.fromDirectory(rootDesignDir);
+        //db.syncDesignDocsWithDb();
```

Comment:
Any reason to keep the call to `db.syncDesignDocsWithDb()` around even as a comment?  The method is being removed as part of this PR, right?  Same for line 94.